pkg/sql/logictest/testdata/logic_test/owner

```diff
@@ -142,7 +142,7 @@ REVOKE GRANT OPTION FOR ALL PRIVILEGES ON DATABASE d FROM testuser2
 user testuser2
 
 # testuser2 has ALL privileges, no grant options, and is not a member of a role,
-# so it can't GRANT.
```

Comment:
new years resolution to eradicate apostrophes?

Reply:
They mess with the quote/unquote matching in any syntax highlighting that thinks these files are SQL, since `#` isn't a valid SQL comment